### PR TITLE
Modify TP init script to be systemd compliant

### PR DESCRIPTION
**Pull request by shamrickus**
This PR modifies the init script to be systemd complaint, which it was before it was rewritten to support `pm2` in #6559
<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Portal
- Automation (CI)

## What is the best way to verify this PR?
Attempt to run the `/etc/init.d/traffic_portal` script as a systemd service and verify that it works.

## If this is a bugfix, which Traffic Control versions contained the bug?

 - master

## PR submission checklist
- [ ] This PR has tests 
- [ ] This PR has documentation 
- [ ] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_portal/build/etc/init.d/traffic_portal [hunk 1]
```diff
@@ -46,14 +62,14 @@ start() {
...
-      --name "$NAME"
+      --name "$NAME" 2>&1 > /dev/null
     RETVAL=$?
 }
 
 stop() {
     if pm2 list | grep -q "$NAME"; then
         echo "Shutting down $NAME"
-        pm2 stop $APPLICATION_PATH
+        pm2 stop $APPLICATION_PATH 2>&1 > /dev/null
```
Comment (zrhoffman) [inline traffic_portal/build/etc/init.d/traffic_portal:72]: `2>&1` needs to go after `> /dev/null` or it doesn't redirect `/dev/stderr` to `/dev/null`

**Review by zrhoffman — CHANGES_REQUESTED** [review]
https://github.com/apache/trafficcontrol/blob/1867d42ac8573826e49405115be0a71802068e3f/traffic_portal/build/etc/init.d/traffic_portal#L69-L70

https://github.com/apache/trafficcontrol/blob/1867d42ac8573826e49405115be0a71802068e3f/traffic_portal/build/etc/init.d/traffic_portal#L85-L86

Since the `pipefail` bash option is not enabled by default, this condition is always met.

File: traffic_portal/build/etc/init.d/traffic_portal (hunk 1 above)
Comment (shamrickus) [reply traffic_portal/build/etc/init.d/traffic_portal:72]: Will change, I was just copy & pasting what the script was doing previously.

**Review by zrhoffman — APPROVED** [review]
Looks good!

**Pull request merged**